src/sentry/static/sentry/app/utils/auth.jsx

```diff
@@ -0,0 +1,16 @@
+import cookies from './cookies';
+
+export default {
+  signOut: function () {
```

Comment:
@benvinegar plz tell me how horrible this is and if there's a better way.

The problem is we can't use an API call here to logout, and we need to leverage CSRF. So I want to just POST a traditional form that contains our `csrftokenmiddleware` key that Django expects. Normally this is just done with `{% csrf_token %}` in a Django form... but here we are.